README.md

```diff
@@ -120,15 +120,15 @@ If an external request triggers a process that requires background tasks to run,
 
 ## Current status and roadmap
 
-- **UPCOMING FEATURES:**: In hopeit.engine 0.3.0 we introduced the ability for hopeit.engine to use different Stream Manager in order to simplify implementations by configuration. We are planning to create plugins to provide a hopeit.engine base Stream Processing engine to replace Redis Streams to simplify operations in some scenarios. Thanks to this, Kafka integration can be supported in the future.
+- **UPCOMING FEATURES:**: `hopeit.engine` 0.3.0 introduced the ability to use different Stream Managers in order and enable custom integrations. We are planning to create plugins to provide a `hopeit.engine` based Stream Processing engine to optionally use instead Redis Streams, simplifying operations in some scenarios. Thanks to this modularization, also Kafka integration can be supported in the future.
```

Comment:
Please use grammarly to correct the sentence, I don´t expect to copy paste the comments as they are, they need also to be reviewed.

Reply:
**UPCOMING FEATURES:**: `hopeit.engine` 0.3.0 introduced the ability to use different Stream Managers and enable custom integrations. We are planning to create plugins to provide a `hopeit.engine` based Stream Processing engine to optionally use instead of Redis Streams, simplifying operations in some scenarios. Thanks to this modularization, also Kafka integration can be supported in the future.